node/core/pvf/src/worker_common.rs

```diff
@@ -187,21 +187,12 @@ where
 			let stream = UnixStream::connect(socket_path).await?;
 			let _ = tokio::fs::remove_file(socket_path).await;
 
-			let result = event_loop(handle.clone(), stream.clone()).await;
-
-			if let Err(err) = stream.shutdown(Shutdown::Both) {
-				// Log, but don't return error here, as it may shadow any error from `event_loop`.
-				gum::debug!(
-					target: LOG_TARGET,
-					"error shutting down stream at path {}: {}",
-					socket_path,
-					err
-				);
-			}
```

Comment:
Removed this because `UnixStream` in `tokio` doesn't implement `Clone`, and I couldn't figure out how to get the stream to stick around here without a really big refactor (or a mutex). After spending an hour on this I just gave up, but it's fine because `shutdown` here was added merely as a courtesy call and shouldn't be necessary. Worst case scenario, the host doesn't get the signal and just waits and times out.